boards/arm/disco_l475_iot1/disco_l475_iot1.dts

```diff
@@ -66,6 +66,14 @@
 	};
 };
 
+&spi1 {
+	status = "ok";
+};
+
+&spi3 {
+	status = "ok";
```

Comment:
Same here, spi3 IRQ pri is defaultly set to 0 instead of 10

Reply:
I configured default SPI IRQ prio to 5 on SOC level.